packages/suite/src/actions/suite/storageActions.ts

```diff
@@ -333,6 +333,7 @@ export const saveMetadata = () => async (_dispatch: Dispatch, getState: GetState
             providers: metadata.providers,
             enabled: metadata.enabled,
             selectedProvider: metadata.selectedProvider,
+            cancelledForDevices: metadata.cancelledForDevices,
```

Comment:
todo: remove

Reply:
removed, or actually saving always `{}`not sure if this is correct but it satysfies typescript. I don't want to make this field optional in redux state. or should it be optional?